.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature request
+    url: https://github.com/openpgpjs/openpgpjs/discussions/categories/ideas
+    about: Suggest an idea for this project
+  - name: Question
+    url: https://github.com/openpgpjs/openpgpjs/discussions
+    about: Please ask and answer questions here
```

This is very nitpicky but when they come here they're probably looking to ask questions, not answer them 🙃 So maybe:
```suggestion
    about: Please ask any questions here

```
Also, please make sure there's a newline at the end of the file (not sure if this suggestion achieves that correctly)